CODEOWNERS

```diff
@@ -3,6 +3,11 @@
 CODEOWNERS @US-Trustee-Program/ustp-team
 LICENSE.md @US-Trustee-Program/ustp-team
 
-*.bicep @US-Trustee-Program/enterprise-architecture
-ops/cloud-deployment @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/network @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/sql @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/keyvault @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/cosmos/cosmos-account.bicep @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/cosmos/cosmos-role-assignment.bicep @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/cosmos/cosmos-custom-role.bicep @US-Trustee-Program/enterprise-architecture
+ops/cloud-deployment/lib/identity @US-Trustee-Program/enterprise-architecture
```

Comment:
Would we also want to add 

ops/cloud-deployment/backend-api-deploy.bicep 
ops/cloud-deployment/frontend-webapp-deploy.bicep
ops/cloud-deployment/main.bicep
ops/cloud-deployment/ustp-cams-cosmos.bicep   

?